src/eip7594/eip7594.c

```diff
@@ -730,6 +816,10 @@ C_KZG_RET verify_cell_kzg_proof_batch(
         if (ret != C_KZG_OK) goto out;
     }
 
+    ////////////////////////////////////////////////////////////////////////////////////////////////
+    // Compute random linear combination of the proofs
+    ////////////////////////////////////////////////////////////////////////////////////////////////
+
     /* Do the linear combination */
```

Comment:
Maybe this comment is a bit redundant now.

Reply:
Thanks! Addressed!